psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
```

Comment:
Guess we need to decide if pasture is (1) purely plugin-like where needn't be built/detected at build time and could instead be loaded at runtime and interacts only with driver code through hooks and no C++ code conditionals or (2) more a conventional addon library that does need to be built/detected at build time and can conditionally build C++ code.

If the latter, can attache a `USING_transqt2ccsort` compile definition to conditionally add `RUN_CCTRANSORT` option to read_options.